src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -173,28 +175,28 @@ export const privateRecordsChoiceHelp = (
       a computer with a fast Internet connection. The digital file could be
       uploaded as a .pdf or other photo file format, like a .jpeg or .png.
     </p>
-    <h4>We get your medical records from your doctor</h4>
+    <h4>We get your medical records for you</h4>
     <p>
-      We can get your medical records for you, but you’ll first need to fill
-      out an Authorization to Disclose Information to VA (VA Form 21-4142) so
-      we can request your records. Getting your records might take us some
-      time, and this could mean it’ll take us longer to make a decision on
-      your claim.
+      If you tell us which VA medical center treated you for your condition, we
+      can get your medical records for you. Getting your records may take us some
+      time, and this could mean that it’ll take us longer to make a decision on
+      your claim. You’ll need to first fill out an Authorization to Disclose
+      Information to the VA (VA Form 21-4142) so we can request your records.
+      You’ll need to fill out a form for each doctor.
     </p>
     <p>
-      <a href="https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf" target="_blank">
+      <a href={vaForm4142URL} target="_blank">
         Download VA Form 21-4142
       </a>.
     </p>
   </AdditionalInfo>
 );
 
-const firstOrNextString = (evidenceTypes) => (evidenceTypes['view:vaMedicalRecords'] ? 'next' : 'first');
+const firstOrNextString = (evidenceTypes) => (evidenceTypes['view:vaMedicalRecords'] ? 'Now' : 'First,');
```

Comment:
You could rename this `const` to `firstOrNowString` if you wanted to keep the semantics. I don't have a strong opinion either way.